### PR TITLE
LFS: miscellaneous changes

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -215,6 +215,7 @@ if not disable_crypt_option
 endif
 
 if not disable_bsd_option
+	rtdl_include_dirs += include_directories('options/bsd/include')
```
Comment (Geertiebear) [inline meson.build:218]: Why does ``rtdl`` need the BSD includes?

Comment (no92) [reply meson.build:218]: I don't recall the exact reason (but there was one), but I suspect it is `unistd.h` pulling in `bits/bsd/bsd_unistd.h` if the BSD option is enabled.

File: options/bsd/include/bits/bsd/bsd_unistd.h [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+extern "C" {
+#endif
+
+void *sbrk(intptr_t increment);
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif // _BSD_UNISTD_H
```
Comment (Dennisbonke) [inline options/bsd/include/bits/bsd/bsd_unistd.h:14]: We should use C style comments here.

File: options/bsd/include/mlibc/bsd-sysdeps.hpp [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#ifndef MLIBC_BSD_SYSDEPS
+#define MLIBC_BSD_SYSDEPS
+
+namespace [[gnu::visibility("hidden")]] mlibc {
+
+[[gnu::weak]] int sys_brk(void **out);
+
+} // namespace mlibc
```
Comment (Dennisbonke) [inline options/bsd/include/mlibc/bsd-sysdeps.hpp:8]: We should use C style comments here.

File: options/bsd/include/mlibc/bsd-sysdeps.hpp [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+#endif // MLIBC_BSD_SYSDEPS
```
Comment (Dennisbonke) [inline options/bsd/include/mlibc/bsd-sysdeps.hpp:10]: We should use C style comments here.

File: options/glibc/include/netinet/icmp6.h [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+#define ND_OPT_REDIRECTED_HEADER 4
+#define ND_OPT_MTU 5
+#define ND_OPT_RTR_ADV_INTERVAL 7
+#define ND_OPT_HOME_AGENT_INFO 8
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif // _NETINET_ICMP6_H
```
Comment (Dennisbonke) [inline options/glibc/include/netinet/icmp6.h:21]: We should use C style comments here.

File: options/internal/include/mlibc/all-sysdeps.hpp [hunk 6]
```diff
@@ -20,6 +20,10 @@
 #	include <mlibc/glibc-sysdeps.hpp>
 #endif /* __MLIBC_GLIBC_OPTION */
 
+#if __MLIBC_BSD_OPTION
+#	include <mlibc/bsd-sysdeps.hpp>
+#endif // __MLIBC_BSD_OPTION
```
Comment (Dennisbonke) [inline options/internal/include/mlibc/all-sysdeps.hpp:25]: We should use C style comments here.

File: sysdeps/linux/generic/sysdeps.cpp [hunk 7]
```diff
@@ -1558,4 +1558,16 @@ int sys_sigtimedwait(const sigset_t *__restrict set, siginfo_t *__restrict info,
...
+int sys_brk(void **out) {
+	auto ret = do_syscall(SYS_brk, 0);
+	if (int e = sc_error(ret); e) {
+		return e;
+	}
+
+	*out = (void *) sc_int_result<uintptr_t>(ret);
+	return 0;
+}
+#endif // __MLIBC_BSD_OPTION
```
Comment (Dennisbonke) [inline sysdeps/linux/generic/sysdeps.cpp:1571]: We should use C style comments here.

File: sysdeps/linux/generic/sysdeps.cpp [hunk 8]
```diff
@@ -1558,4 +1558,16 @@ int sys_sigtimedwait(const sigset_t *__restrict set, siginfo_t *__restrict info,
 	return 0;
 }
 
+#if __MLIBC_BSD_OPTION
+int sys_brk(void **out) {
+	auto ret = do_syscall(SYS_brk, 0);
+	if (int e = sc_error(ret); e) {
```
Comment (Dennisbonke) [inline sysdeps/linux/generic/sysdeps.cpp:1564]: We don't do spaces after `if`.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
A few styling issues, otherwise LGTM.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**